src/ocm/components/clusterWizard/wizardTransition.ts

```diff
@@ -53,7 +54,10 @@ const staticIpValidationsMap: WizardStepValidationMap = {
     validationIds: [],
   },
   softValidationIds: [],
-};
+});
+
+const staticIpValidationsMap = buildEmptyValidationsMap();
+const operatorsStepValidationsMap = buildEmptyValidationsMap();
```

Comment:
There are certain operators related validations which should be IMHO included in operator's step validations map:

Host validations:
https://github.com/openshift-assisted/assisted-ui-lib/blob/master/src/common/api/types.ts#L1279-L1283

Cluster validations:
https://github.com/openshift-assisted/assisted-ui-lib/blob/master/src/common/api/types.ts#L516-L519

Reply:
Sorry, I missed this.
The problem if we add the validations in this page, is that the users won't know what the actual issue is.

In Host Discovery, they are able to see which requirements are unfulfilled for each host, and they can figure out what they need to do, but not on Operators step.

Reply:
These cluster and hosts validation will pop out only after the hosts started to show up during the discovery step. 
Therefore we cannot move the validations from the discovery step.